pkg/KubeArmorAnnotation/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
refer to https://github.com/kubearmor/KubeArmor/blob/main/pkg/KubeArmorPolicy/hack/boilerplate.go.txt